app/views/bookings/index.html.erb

```diff
@@ -1,2 +1,5 @@
 <h1>Bookings#index</h1>
 <p>Find me in app/views/bookings/index.html.erb</p>
+<% @bookings.each do |booking| %>
+<h1>user_id: <%= booking.user_id %> </h1>
```

Comment:
Can we add a bit more complete information here, and link to each booking?

Reply:
Ive added a bit more data to the view page